### DISCUSSION TITLE
TeX Studio 相关讨论

### DISCUSSION THREAD

**Discussion by TheNetAdmin** [TheNetAdmin/zjuthesis#98] [Tool Usage]
请在这里讨论 TeXStudio 相关问题，例如 TeXStudio 如何编译参考文献

**Comment by FoolgryGamer** [comment]
TeXStudio 如何编译参考文献，在ref.lib中添加了Bibtex格式的引用，但是没有相应参考文献引出（纯小白）

**Reply by TheNetAdmin** [reply]
#23

**Comment by muzimuzhi** [comment]
texstudio 下使用 latexmk：
 - 约定
   - 默认配置窗口左下角的 "Show Advanced Options" 为勾选状态
 - 编译
   - [最小修改] 配置窗口 -> Commands tab -> 找到 latexmk 一项，把它从 `latexmk -pdf -silent -synctex=1 %` 修改为 `latexmk %`。通过菜单项 Tools -> Commands -> Latexmk 执行编译。
   - [可选] 不修改 texstudio 预设 commands 里的 latexmk，自行定义用户 command；配置快捷键；把菜单项添加到工具栏。详见文章《[[LaTeX 编辑器] 在 TeXstudio 里自定义一个工具栏按钮](https://zhuanlan.zhihu.com/p/60996450)》
 - 查看
   - [最小修改] 配置窗口 -> Build tab -> 在底部找到 "Additional Search Paths"，在 "Log File" 和 "PDF File" 里都填入 `out`。参考 [Using a separate directory for auxiliary files - TeXstudio wiki](https://github.com/texstudio-org/texstudio/wiki/Tips-And-Tricks#using-a-separate-directory-for-auxiliary-files)

**Reply by zjuKcheng** [reply]
解决了，膜拜~

**Reply by JK-Oliver** [reply]
使用内置PDF查看器查看并 热刷新
latexmk % | txs:///view-pdf-internal --embedded

**Reply by Luorifeihua** [reply]
如果没找到additional search path，需要在选项-->设置TeXstudio（配置窗口）左下角“显示高级选项”一栏勾选

**Comment by FoolgryGamer** [comment]
***@***.***

**Comment by FoolgryGamer** [comment]
***@***.***
